File: src/main/java/no/nav/veilarbaktivitet/aktivitetskort/idmapping/IdMappingDto.java
package no.nav.veilarbaktivitet.aktivitetskort.idmapping;


import no.nav.veilarbaktivitet.arena.model.ArenaId;

import java.util.UUID;

public record IdMappingDto(
    ArenaId arenaId,
    Long aktivitetId,
    UUID funksjonellId
) {

    public static IdMappingDto map(IdMapping idMapping) {
        return new IdMappingDto(idMapping.arenaId(), idMapping.aktivitetId(), idMapping.funksjonellId());
    }
}

File: src/main/kotlin/no/nav/veilarbaktivitet/aktivitetskort/idmapping/IdMappingDto.kt
package no.nav.veilarbaktivitet.aktivitetskort.idmapping

import no.nav.veilarbaktivitet.arena.model.ArenaId
import java.util.*

data class IdMappingDto(val arenaId: ArenaId, val aktivitetId: Long, val funksjonellId: UUID) {
    companion object {
        fun map(idMapping: IdMapping): IdMappingDto {
            return IdMappingDto(idMapping.arenaId, idMapping.aktivitetId, idMapping.funksjonellId)
        }
    }
}
